airwallex/src/main/java/com/airwallex/android/AirwallexStarter.kt

```diff
@@ -31,10 +33,36 @@ class AirwallexStarter {
         fun initialize(
             application: Application,
             configuration: AirwallexConfiguration,
-            clientSecretProvider: ClientSecretProvider? = null
         ) {
             AirwallexActivityLaunch.initialize(application)
-            Airwallex.initialize(application, configuration, clientSecretProvider)
+            Airwallex.initialize(application, configuration)
+        }
+
+        /**
+         * Launch the card payment flow to allow the user to complete the entire payment flow
+         *
+         * @param activity activity {@link Activity}
```

Comment:
Shall we improve this by `the host activity on which the payment UI is presented`?

Reply:
ok